processor/k8sattributesprocessor/config.go

```diff
@@ -123,7 +123,7 @@ type ExtractConfig struct {
 //	    extract:
 //	    labels:
 //
-//	  - name: $1
+//	  - tag_name: $$1
```

Comment:
Why `$$` needed?

Reply:
Beacause `$1` will be treat as environment 1, see [doc](https://opentelemetry.io/docs/collector/configuration/#configuration-environment-variables)